app/src/main/java/com/example/giftly/CThemeScreen.java

```diff
@@ -15,88 +15,100 @@
+            Button etrThemeBtn = findViewById(R.id.easterTheme);
+            Button savingit = findViewById(R.id.saveIt);
+
+            savingit.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    Intent intent = new Intent(CThemeScreen.this, HomeScreen.class);
+                    startActivity(intent);
+                }
+            });
+
+            dftThemeBtn.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    int selectedColor = ContextCompat.getColor(CThemeScreen.this, R.color.Default_color);
+                    sharedPreferences.edit().putInt("BackgroundColor", selectedColor).apply();
+                    themeSubject.notifyObservers(selectedColor);
+                }
+            });
+
+            glyThemeBtn.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    int selectedColor = ContextCompat.getColor(CThemeScreen.this, R.color.Giftly_color);
+                    sharedPreferences.edit().putInt("BackgroundColor", selectedColor).apply();
+                    themeSubject.notifyObservers(selectedColor);
+                }
+            });
+            //Valentines
+            vltThemeBtn.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    int selectedColor = ContextCompat.getColor(CThemeScreen.this, R.color.Valentines_color);
+                    sharedPreferences.edit().putInt("BackgroundColor", selectedColor).apply();
+                    themeSubject.notifyObservers(selectedColor);
+                }
+            });
+
+            etrThemeBtn.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    int selectedColor = ContextCompat.getColor(CThemeScreen.this, R.color.Easter_color);
+                    sharedPreferences.edit().putInt("BackgroundColor", selectedColor).apply();
+                    themeSubject.notifyObservers(selectedColor);
+                }
+            });
+        }
+
+        @Override
+        public void onThemeChanged(int newColor) {
+            getWindow().getDecorView().setBackgroundColor(newColor);
+        }
+}
+
+
+    //Back button
+    //ActionBar actionBar = getSupportActionBar();
+    //actionBar.setDisplayHomeAsUpEnabled(true);
+
+
 
     //Back button configuration
-    @Override
+   /* @Override
     public boolean onOptionsItemSelected(@NonNull MenuItem item) {
         switch (item.getItemId()) {
             case android.R.id.home:
                 this.finish();
                 return true;
         }
         return super.onOptionsItemSelected(item);
-    }
-}
\ No newline at end of file
+}
+*/
```

Comment:
Maybe delete the commented back button configuration if its not going to be used.